### PR TITLE
BP-7 | Don’t kick active clients

### PR DESCRIPTION
**Pull request by Keksgesicht**
As a stakeholder, we want that only inactive clients get disconnected from any Access Point.

Acceptance Criteria:
- Only kick clients which have a data rate of <50 kbit/s withing the last 30 seconds (VOIP calls!).
- So if 50kbit/s is transmitted in TX or RX direction, the client should not be disconnected.
- 50 kbit/s should be the default value. Make a config switch.

### CODE REVIEW

**Comment by Keksgesicht** [comment]
commit 10427e8b1c21651c1f7a616e53a1139a380770dc
I could have used realloc() but
- lowring the size
  - index now outside the array
  - throwing away data of multiple seconds
- raising the size
  - now having random or null values

changing size again and again during runtime not typical. So just wait sometime until the old clients disappear.

File: main.c [hunk 1]
```diff
@@ -92,6 +92,9 @@ void usteer_init_defaults(void)
 	config.load_kick_min_clients = 10;
 	config.load_kick_reason_code = 5; /* WLAN_REASON_DISASSOC_AP_BUSY */
 
+	config.kick_client_active_sec = 30;
```
Comment (PhilipJonasFranz) [inline main.c:95]: Measure in Bit/s, not KBit/s, replace with 50 * 1000;

**Review by Peter-Nicomann — APPROVED** [review]
Programm Läuft, verbraucht nicht 90% des Rams und ist auch nach Commit regeln Commited.

**Review by Peter-Nicomann — APPROVED** [review]
gut

**Pull request merged**